docs/src/main/sphinx/connector/jdbc-authentication.fragment

```diff
@@ -0,0 +1,55 @@
+Data source authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector can provide credentials for the data source connection
+in following ways:
+
+* Inline, in the connector configuration file.
+* In a separate properties file.
+* In a key store file.
+* As extra credentials set when connecting to Trino.
+
+You can use :doc:`secrets </security/secrets>` to avoid storing sensitive
+values in the catalog properties files.
+
+The following table describes configuration properties
+for connection credentials:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Property name
+    - Description
+  * - ``credential-provider.type``
+    - Type of the credential provider. Must be one of ``INLINE``,
+      ``FILE``, or ``KEYSTORE``; defaults to ``INLINE``.
+  * - ``connection-user``
+    - Connection user name.
+  * - ``connection-password``
+    - Connection password.
+  * - ``user-credential-name``
+    - Name of the extra credentials property, which value to use
+      as the user name. See ``extraCredentials``
+      in :ref:`jdbc-parameter-reference`.
+  * - ``password-credential-name``
+    - Name of the extra credentials property, which value to use
+      as the password.
```

Comment:
```suggestion
    - Name of the extra credentials property, whose value to use
      as the user name. See ``extraCredentials``
      in :ref:`jdbc-parameter-reference`.
  * - ``password-credential-name``
    - Name of the extra credentials property, whose value to use
      as the password.
```

Reply:
@mosabua @Jessie212 WDYT? I had to google this and couldn't find a definitive answer.

Reply:
That change is correct.

Reply:
That = current, or suggested?

Reply:
It sounds better to say: 

... whose value is used as the username.
... whose value is used as the password.

Reply:
Thanks, updated!